### ISSUE TITLE
Find some bugs 

### ISSUE THREAD

**Issue by JinXuekun** [AI4Finance-LLC/ElegantRL#25]
Thanks for your wonderful work!

I found 2 bugs from the repo.

1. In [https://github.com/AI4Finance-LLC/ElegantRL/blob/master/elegantrl/agent.py](url) and other `agent.py` files, 
    method `get_obj_critic_per()` may compute the td-error wrong.

    ` td_error = (q_label - torch.min(q1, q1).detach()).abs()`
      should be 
    ` td_error = (q_label - torch.min(q1, q2).detach()).abs()`

2. Also in `agent.py`:  class `AgentSAC`,`AgentModSAC`, and `AgentSharedSAC`.
    
    Their method `update_net()` return the following

    `return alpha.item(), obj_critic.item()`

     It may be 

    `return obj_actor.item(), obj_critic.item()`

**Comment by 891922758** [comment]
Yeah. Thank you for your kind reminding. The first one is a bug. The second one is actually a small trick we may use internally and we changed it to the standard version in the repo.
